### ISSUE TITLE
Does not work with Powershell Core

### ISSUE THREAD

**Issue by oledid** [DavidAnson/ConvertTo-Jpeg#13]
https://devblogs.microsoft.com/scripting/powertip-identify-if-you-are-running-on-powershell-core/

**Comment by DavidAnson** [comment]
Isn't this more of an issue with PowerShell Core?

**Comment by oledid** [comment]
Yeah maybe, but it could display a better error if the referenced
assemblies fails to load, perhaps?

**Comment by oledid** [comment]
Yeah maybe, but it could display a better error if the referenced assemblies fails to load, perhaps?

**Comment by jamacoe** [comment]
I found a similar thread, someone saying "WinRT support was removed from .Net 6.0 and as result from PowerShell 7.2 too.". David, any chance you will adapt this valuable script to PS Core?

**Comment by DavidAnson** [comment]
I don't use this script anymore, so the urgency for me is less. This issue is a candidate for PR if someone digs in and figures out the changes!
